Terminal.Gui/Text/TextFormatter.cs

```diff
@@ -1226,6 +1328,30 @@ public Size GetFormattedSize ()
 		/// </remarks>
 		public bool NeedsFormat { get; set; }
 
+		/// <summary>
+		/// Gets or sets the number of columns used for a tab.
+		/// </summary>
+		public int TabWidth {
+			get => _tabWidth;
+			set => _tabWidth = EnableNeedsFormat (value);
+		}
+
+		/// <summary>
+		/// Gets or sets a value indicating whether multi line is allowed.
```

Comment:
Same question: What is the behavior if this is false? We should clearly document that.

Reply:
I'll look better on the documents, which you know I'm weak :-)